README.md

```diff
@@ -1,50 +1,1323 @@
+   | Cisco ACI plugin | admin            | Plug!n12$4       | Fabric:XAuthToken:ACI_v1.0.0 |
+   
+   Use the following curl command to add the plugin:
+   
+   ```
+   curl -i POST \
+       -H 'Authorization:Basic {base64_encoded_string_of_<odim_username:odim_password>}' \
+       -H "Content-Type:application/json" \
+       -d \
+    '{"HostName":"{plugin_host}:{port}",
+      "UserName":"{plugin_userName}",
+      "Password":"{plugin_password}", 
+      "Links":{
+          "ConnectionMethod": {
+             "@odata.id": "/redfish/v1/AggregationService/ConnectionMethods/{ConnectionMethodId}"
+          }
+       }
+    }' \
+     'https://{odim_host}:30080/redfish/v1/AggregationService/AggregationSources' -kNOTE: To generate a base64 encoded string of `{odim_username:odim_password}`, run the following command:
+   ```
+   
+   <blockquote>
+       NOTE: To generate a base64 encoded string of `{odim_username:odim_password}`, run the following command:</blockquote>
+   
+   ```
+   $ echo -n '{odim_username}:{odim_password}' | base64 -w0
+   ```
+   
+   Replace `{base64_encoded_string_of_[odim_username:odim_password]}` with the generated base64 encoded string in the curl command. You will receive:
+   
+      -   An HTTP `202 Accepted` status code.
+      -   A link to the task monitor associated with this operation in the response header.
+   
+      To know the status of this task, perform HTTP `GET` on the `taskmon` URI until the task is complete. If the plugin is added successfully, you will receive an HTTP `200 OK` status code.
+   
+   After the plugin is successfully added, it will also be available as a manager resource at:
+   
+      `/redfish/v1/Managers`
+   
+   For more information, refer to "Adding a plugin" in the [Resource Aggregator for Open Distributed Infrastructure Management™ API Reference and User Guide](https://github.com/ODIM-Project/ODIM/tree/development/docs).  
+   
+2. To verify that the added plugin is active and running, do the following: 
+
+   1. To get the list of all available managers, perform HTTP `GET` on: 
+
+      `/redfish/v1/Managers` 
+
+      You will receive JSON response having a collection of links to the manager resources. You will see the following links in the collection:
+
+      -   A link to the resource aggregator manager.
+
+      -   Links to all the added plugin managers.
+
+   2. To identify the plugin Id of the added plugin, perform HTTP `GET` on each manager link in the response. 
+      The JSON response body for a plugin manager has `Name` as the plugin name.
+      Example:
+      The JSON response body for the Cisco ACI plugin manager has `Name` as `CiscoACI`.
+
+      **Sample response**
+
+      ```
+      {
+              "@odata.context":"/redfish/v1/$metadata#Manager.Manager",
+              "@odata.etag":"W/\"AA6D42B0\"",
+              "@odata.id":"/redfish/v1/Managers/536cee48-84b2-43dd-b6e2-2459ac0eeac6",
```

Comment:
sample response is:

{
    "@odata.context": "/redfish/v1/$metadata#Manager.Manager",
    "@odata.id": "/redfish/v1/Managers/fb40f2dc-0c6d-4464-bc98-fea775adbbb9",
    "@odata.type": "#Manager.v1_10_0.Manager",
    "FirmwareVersion": "v1.0.0",
    "Id": "fb40f2dc-0c6d-4464-bc98-fea775adbbb9",
    "Links": {
        "ManagerForSwitches": [
            {
                "@odata.id": "/redfish/v1/Fabrics/fb40f2dc-0c6d-4464-bc98-fea775adbbb9:1/Switches/af10c386-68d5-45aa-b3c3-431e3e4c3647:101"
            },
            {
                "@odata.id": "/redfish/v1/Fabrics/fb40f2dc-0c6d-4464-bc98-fea775adbbb9:1/Switches/668f20cf-b6e7-4ded-a180-bf8e33dc18fc:102"
            },
            {
                "@odata.id": "/redfish/v1/Fabrics/fb40f2dc-0c6d-4464-bc98-fea775adbbb9:1/Switches/7d5a25b3-3ac4-49f4-a929-243b5b97bba0:201"
            }
        ],
        "ManagerForSwitches@odata.count": 3
    },
    "ManagerType": "Service",
    "Name": "ACI",
    "Status": {
        "Health": "OK",
        "State": "Enabled"
    },
    "UUID": "fb40f2dc-0c6d-4464-bc98-fea775adbbb9"
}